com.avaloq.tools.ddk.xtext.builder/src/com/avaloq/tools/ddk/xtext/builder/MonitoredClusteringBuilderState.java

```diff
@@ -758,17 +758,12 @@ protected void awaitBinaryStorageExecutorQuiescence() {
    * Log the first and last 10 StackOverflowError Stack Trace.
    *
    * @param ex
-   *          the StackOverflowError
+   *          the StackOverflowError; may get modified
    * @param binding
    *          the object to be inserted into the message
    */
   protected void logStackOverflowErrorStackTrace(final StackOverflowError ex, final URI binding) {

```

Comment:
I would remove the method and move the code to the exception handler, then it is clear that modifying the exception is not an issue because after LOGGER.warn it will go out of scope

Reply:
Done.